src/models/GllobalValidations/DescriptionValidation.tsx

```diff
@@ -114,7 +114,7 @@ export const descriptionValidation = {
   ],
   RegionEmail: [
     {
-      pattern: /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
+      pattern: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```

Comment:
type:"email"?

Or is it also laggy

Reply:
> `type:"email"`?
> 
> Or is it also laggy

If "type: "email"" is added, there are type-checking errors, I guess it's a different implementation.